backend/src/main/java/com/relay/controllers/StationController.java

```diff
@@ -1,43 +1,34 @@
 package com.relay.controllers;
 
-import java.util.List;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RestController;
 
-import com.relay.model.Relay;
-import com.relay.model.places.Station;
-import com.relay.service.StationService;
-
 @RestController
 public class StationController {
 
-    private final StationService stationService;
-
-    @Autowired
-    public StationController(StationService stationService) {
-
-        this.stationService = stationService;
-    }
-
-    @GetMapping("/stations/{id}/relays")
-    public List<Relay> retrieveAllRelaysOnStation(@PathVariable Long id) {
-
-        Station stationOptional = stationService.findOne(id);
-
-        // if (!stationOptional.isPresent()) {
-        // throw new RelayNotFoundException("id- " + id);
-        // }
-
-        return stationOptional.getRelay();
-    }
-
-    @GetMapping("/stations")
-    public List<Station> retrieveAllStation() {
-
-        return stationService.findAll();
-    }
+    // private final StationService stationService;
+    //
+    // @Autowired
+    // public StationController(StationService stationService) {
+    //
+    // this.stationService = stationService;
+    // }
+    //
+    // @GetMapping("/stations/{id}/relays")
+    // public List<Relay> retrieveAllRelaysOnStation(@PathVariable Long id) {
+    //
+    // Station stationOptional = stationService.findOne(id);
+    //
+    // // if (!stationOptional.isPresent()) {
+    // // throw new RelayNotFoundException("id- " + id);
+    // // }
+    //
+    // return stationOptional.getRelay();
+    // }
+    //
+    // @GetMapping("/stations")
+    // public List<Station> retrieveAllStation() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)